[Java] 3.64.0 Release Notes (#696)
SAP/cloud-sdk: docs/java/release-notes.mdx

### Fixed Issues
-->

## 3.64.0 - February 24, 2022

[Maven Central](https://search.maven.org/search?q=g:com.sap.cloud.sdk*%20AND%20v:3.64.0) | [Javadoc](https://help.sap.com/doc/55bb1642ea614bf582d514ed34401136/1.0/en-US/index.html)

### New Functionality

> - Extend the implementation for using the BTP Destination & Connectivity services to now also support the `x509` authentication type.

Reviewer: ```suggestion
- The **Destination** and  **Connectivity** services on SAP BTP now also support the `x509` authentication type.
```
